pkg/codegen/dotnet/doc.go

```diff
@@ -43,6 +43,15 @@ func (d DocLanguageHelper) GetDocLinkForResourceType(packageName, _, typeName st
 	return fmt.Sprintf("/docs/reference/pkg/dotnet/Pulumi%s/%s.html", packageNamespace, typeName)
 }
 
+// GetDocLinkForBuiltInType returns the godoc URL for a built-in type.
+// Currently not using the typeName parameter because the returned link takes to a general page
```

Comment:
Yep this is why I hard-coded the link and postponed the decision on how far we want to take the linking. I suggest adjusting the comment, though, to just say that it returns a link to the top-level built-in types page regardless of the type, and just leave it at that.

Reply:
Agreed!